src/internal/utils/use-auto-scroll.ts

```diff
@@ -68,7 +68,7 @@ export function useAutoScroll() {
           document.activeElement === activeElementBeforeDelay &&
           getLastInteraction() === "keyboard"
         ) {
-          document.activeElement.scrollIntoView({ behavior: "smooth", block: "nearest" });
+          document.activeElement.scrollIntoView?.({ behavior: "smooth", block: "nearest" });
```

Comment:
This method is not supported in JSDOM. When added more test cases in this PR, they started visiting this line and crashed.

We do the same in the main components repo already

https://github.com/cloudscape-design/components/blob/5f488fdbf4086072850e00988889ef9358eb4945/src/internal/utils/scrollable-containers.ts#L66-L81